collector/textfile.go

```diff
@@ -249,3 +247,15 @@ fileLoop:
 	)
 	return nil
 }
+
+// hasTimestamps returns true when metrics contain unsupported timestamps.
+func hasTimestamps(parsedFamilies map[string]*dto.MetricFamily, path string) bool {
```

Comment:
One last thing: you are passing `path` into this function, but the function doesn't do anything with it (anymore). Could you remove that argument?

Reply:
@juliusv yop lgtm